src/api/sync.rs

```diff
@@ -501,12 +507,69 @@ impl ApiRepo{
     /// api.model("gpt2".to_string()).info();
     /// ```
     pub fn info(&self) -> Result<RepoInfo, ApiError> {
-        let url = format!("{}/api/{}", self.api.endpoint, self.repo.api_url());
-        let response = self.api.client.get(&url).call().map_err(Box::new)?;
+        Ok(serde_json::from_value(self.detailed_info().send()?)?)
+    }
+
+    /// Get a builder to help create a request to the hub API
+    /// ```
+    /// # use hf_hub::api::tokio::Api;
+    /// # tokio_test::block_on(async {
+    /// let api = Api::new().unwrap();
+    /// api.model("gpt2".to_owned())
+    ///     .detailed_info()
+    ///     .query("blobs".to_owned(), "true".to_owned())
+    ///     .send();
+    /// # })
+    /// ```
+    pub fn detailed_info(&self) -> DetailedInfoBuilder {
+        DetailedInfoBuilder::new(self.api.clone(), &self.api.endpoint, &self.repo)
```

Comment:
```suggestion
        DetailedInfoBuilder::new(self.api.clone(), &self.repo)
```
endpoint is already in api, no need to duplicate.

Reply:
Actually use `ApiRepo` instead so you need just Self.